package.json

```diff
@@ -21,10 +21,15 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "lint": "eslint .",
```

Comment:
Good point, do you prefer to have an `.eslintignore` file or simplu use the `--ignore-pattern` flag inside the `lint` script now that is decoupled from the rest?